src/test/resources/settings.json

```diff
@@ -21,6 +21,15 @@
       "capabilities": {
         "deviceName": "Android Emulator"
       }
+    },
+    "ios": {
+      "applicationPath": "./src/test/resources/apps/TestApp.app.zip",
+      "capabilities": {
+        "deviceName": "iOS Simulator",
+        "automationName": "XCUITest",
+        "platformName": "iOS",
```

Comment:
are all these attributes required?

Reply:
yes, default driver is UIAutomation (should be fixed in Appium official document too).